summary_workflows/quantification/specification/Q_benchmark_specification.md

```diff
@@ -125,27 +166,39 @@ The OpenEBench assessment file contains the following attributes:
 - **participant_id** - benchmarked tool
-- **stderr** - standard error, set to zero
-- **value** - metric value
-- **metric_id** - metric name; metric names used in this benchmark are specified in the table below
+- **metrics**:
+	- **value** - metric value
+	- **metric_id** - metric name; metric names used in this benchmark are specified in the table below
  
-The following tables list the metric names, value types and units, and a description of each attribute-value pair:
+The following tables list the metric names, value types and units, and a description:
 
 | Metric_id | Type | Unit | Description |
 | :--- | :--- | :--- | :----------------- |
 | `Correlation_coefficient_10nt` | `float` | N/A | Correlation between poly(A) site quantification by benchmarked tool and ground truth dataset calculated for 10 nt distance threshold (window) |
 | `Correlation_coefficient_50nt` | `float` | N/A | Correlation between poly(A) site quantification by benchmarked tool and ground truth dataset calculated for 50 nt distance threshold (window) |
 | `Correlation_coefficient_100nt` | `float` | N/A | Correlation between poly(A) site quantification by benchmarked tool and ground truth dataset calculated for 100 nt distance threshold (window) |
+| `Expression_non-matched-PAS_10nt` | `float` | N/A | Total expression of non-matched sites quantified by benchmarked tool for 10 nt distance threshold (window) |
+| `Expression_non-matched-PAS_50nt` | `float` | N/A | Total expression of non-matched sites quantified by benchmarked tool for 50 nt distance threshold (window) |
+| `Expression_non-matched-PAS_100nt` | `float` | N/A | Total expression of non-matched sites quantified by benchmarked tool for 100 nt distance threshold (window) |
+| `MSE_100nt` | `float` | N/A | Mean Squared Error (MSE) of relative PAS usage calculated from RNAseq-based PAS quantification and orthogonal 3'end seq data for 100 nt distance threshold (window) |
 
 #### Output 2
 
-The OpenEBench consolidation file specifies visualization types (2D scatter plot, barplot), descriptions of metrics used for X and Y axis and (X,Y) value pairs for each challenge.
+The OpenEBench aggregation file contains information required to produce one plot.
+
+#### Output 3
+
+The OpenEBench consolidation file contains all the information about the new benchmarking run and specifies visualization types (2D scatter plot, barplot), descriptions of metrics used for X and Y axis and (X,Y) value pairs for each challenge and challenge participants.
 
 
 [//]: # (References)
  
 [in1]: ./example_files/input1.bed
 [in2]: ./example_files/input2.bed
+[in3]: ./example_files/input3.gtf
 [assessment_out]: ./example_files/assessment_out.json
+[aggregation_out]: ./example_files/aggregation_out.json
+[consolidation_out]: ./example_files/consolidation_out.json
```

Comment:
This file doesn't exist

Reply:
I removed the consolidation file example.